### PR TITLE
Update beta.3 Core docs - Closes #889 

### LINKED ISSUE #899: Updates for Core 3.0.0-beta.3 docs
- Bump version number
- Update forging guide

### PR DESCRIPTION
**Pull request by Tschakki**
Closes #899

### CODE REVIEW

File: modules/ROOT/pages/management/forging.adoc [hunk 1]
```diff
@@ -6,66 +6,28 @@ Mona Bärenfänger <mona@lightcurve.io>
...
-<1> Enter the secret passphrase here that needs to be encrypted.
-<2> Enter the password here that will be required to decrypt the passphrase again.
+== Adding the delegate info to the config
 
-* Type in the passphrase followed by the password required for encryption.
-* This will result in the creation of an `encryptedPassphrase` key-value pair.
+Generate the necessary config object conveniently with the Lisk Core CLI:
 
-== Generating the hash onion
+The following command will return a config object in the console.
```
Comment (ManuGowda) [inline modules/ROOT/pages/management/forging.adoc]:
```suggestion
The following command will return a config object to the console.
```

File: modules/ROOT/pages/management/forging.adoc [hunk 2]
```diff
@@ -6,66 +6,28 @@ Mona Bärenfänger <mona@lightcurve.io>
...
+The object can be copy-pasted directly into the application configuration liek described below.
```
Comment (ManuGowda) [inline modules/ROOT/pages/management/forging.adoc]:
```suggestion
The object can be copy-pasted directly into the application configuration like described below.
```

File: modules/ROOT/pages/management/pm2.adoc [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+We recommend using PM2 to manage the instance.
+
+== Quick setup
+
+[source,bash]
+----
+# Install pm2 globally
+npm i -g pm2
+# Start the node with pm2 under the alias 'lisk-core'
+pm2 start "lisk-core start --network betanet --api-ipc" --name 'lisk-core'
```
Comment (ManuGowda) [inline modules/ROOT/pages/management/pm2.adoc]: what does `--api-ipc` mean here? there is no such flag to my understanding, maybe you meant `--enable-ipc`?

Comment (ManuGowda) [reply modules/ROOT/pages/management/pm2.adoc]: also, always try to run the command to verify, that should give you an error in this case.

File: modules/ROOT/pages/reference/cli.adoc [hunk 4]
```diff
@@ -204,6 +204,40 @@ USAGE
 COMMANDS
   forging:disable  Disable forging for given delegate address.
   forging:enable   Enable forging for given delegate address.
+  forging:config   Generate the forging config object for given delegate address.
```
Comment (ManuGowda) [inline modules/ROOT/pages/reference/cli.adoc]:
```suggestion
  forging:config   Generate delegate forging config for given passphrase and password.
```

File: modules/ROOT/pages/reference/cli.adoc [hunk 5]
```diff
@@ -204,6 +204,40 @@ USAGE
...
+  forging:status   Get forger info for given delegate address.
```
Comment (ManuGowda) [inline modules/ROOT/pages/reference/cli.adoc]:
```suggestion
  forging:status   Get forging information for the locally running node.
```

File: modules/ROOT/pages/reference/cli.adoc [hunk 6]
```diff
@@ -204,6 +204,40 @@ USAGE
...
+                               	- --password=pass:password123 (should only be used where security is not important)
+
+  --pretty                     Prints JSON in pretty format rather than condensed.
+
+EXAMPLES
+  forging:config
+  forging:config --password your_password
+  forging:config --passphrase your_passphrase --password your_password --pretty
+  forging:config --count=1000000 --distance=2000 --output /tmp/forging_config.json
 ----
```
Comment (ManuGowda) [inline modules/ROOT/pages/reference/cli.adoc:242]: The forging:status command seems to be missing?

Comment (Tschakki) [reply modules/ROOT/pages/reference/cli.adoc:242]: it's coming in the next section

File: modules/ROOT/pages/management/pm2.adoc (hunk 3 above)
Comment (Tschakki) [reply modules/ROOT/pages/management/pm2.adoc]: good catch, will update 👍

**Review by ManuGowda — APPROVED** [review]

**Pull request merged**